awx/ui_next/src/screens/Organization/Organization.jsx

```diff
@@ -41,8 +41,10 @@ class Organization extends Component {
   }
 
   async componentDidUpdate(prevProps) {
-    const { location } = this.props;
-    if (location !== prevProps.location) {
+    const { location, match } = this.props;
+    const url = `/organizations/${match.params.id}/`;
+    const shouldUpdate = prevProps.location.pathname.includes(url);
+    if (location !== prevProps.location && shouldUpdate) {
```

Comment:
This is looking much better :+1: what's going on here is we're now checking to make sure that we're navigating _to_ our destination (whichever tab that might be) _from_ another tabbed view for that org. 

@AlexSCorey and I talked about this offline, and I think twhat we have here will prevent the duplicate request when loading the page from the orgs list (as an example), but I don't think it'll prevent the request when we're, say, moving from the _access_ tab to the _teams_ tab.

The lose set of rules we came up with for when this request should fire were:

we _only_ want to make the api request  if:
- we're navigating _to_ the details view specifically
- _and_ the previous view we were on was another tabbed view for that org

for _all_ other cases, we'd expect `loadOrganizationAndRoles` to be called and to have us covered. 

@mabashian thoughts on this? I think this makes sense but I want to make sure I'm following the problem completely.

Reply:
👍 from me